src/Compilers/Core/Portable/DiagnosticAnalyzer/DiagnosticAnalysisContext.cs

```diff
@@ -1622,4 +1641,93 @@ public void ReportSuppression(Suppression suppression)
         /// </summary>
         public SemanticModel GetSemanticModel(SyntaxTree syntaxTree) => _getSemanticModel(syntaxTree);
     }
+
+    /// <summary>
```

Comment:
Can all the changes in this file above this line be reverted? Seems they are not required for this PR.